src/query/expression/src/types/bitmap.rs

```diff
@@ -175,3 +178,15 @@ impl ArgType for BitmapType {
         BinaryColumnBuilder::with_capacity(capacity, 0)
     }
 }
+
+pub fn deserialize_bitmap(buf: &[u8]) -> databend_common_exception::Result<RoaringTreemap> {
```

Comment:
better to move this function to https://github.com/datafuselabs/databend/blob/main/src/common/io/src/bitmap.rs

Reply:
done